week4/spiral/spiral.hs

```diff
@@ -0,0 +1,60 @@
+module Spiral where
+
+import Data.Matrix
+
+data Direction = R | D | U | L
+
+main :: IO ()
+main = do
+    putStrLn("Please enter a size for the spiral!")
+    i <- getInt
```

Comment:
You can avoid writing "getInt" function. You could do:
`i <- read <$> getLine`
The compiler will infer the type of `read` by how the value is used down the line.